ExampleMod/Content/NPCs/ExamplePerson.cs

```diff
@@ -345,14 +343,16 @@ public class ExamplePersonProfile : ITownNPCProfile
 		public int RollVariation() => 0;
 		public string GetNameForVariant(NPC npc) => npc.getNewNPCName();
 
+		public Asset<Texture2D> GetTextureNPCShouldUse() => ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson", AssetRequestMode.ImmediateLoad);
+
```

Comment:
```suggestion
		public Asset<Texture2D> GetTextureNPCShouldUse() {
			return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson", AssetRequestMode.ImmediateLoad);
		}

```

Reply:
funny story, I had actually meant to remove this, since there's no longer a parameterless overload of `GetTextureNPCShouldUse` in `ITownNPCProfile`, but I must've forgotten to. thanks for pointin' this out